F-FrontEnd/src/fincludes/iso_c_binding.f90

```diff
@@ -0,0 +1,195 @@
+module iso_c_binding
+    implicit none
+
+    ! TODO detect architecture and set correct value
+    integer, parameter :: POINTER_LEN = 4 ! 32 bit
```

Comment:
When a module information file (`.mod`) is provided by a user or the backend compiler, Omni must follow the information in it. So we have developed a tool that converts a `.mod` file into `.xmod` for the Fujitsu compiler and gfortran. (This tool is enabled with the `--enable-mod2xmod` option at configuration.) I think this should be the basic way for Omni to process a module without its source. One problem is that there is no `iso_c_binding.mod` found in the gfortran environment because that module is internally processed by gfortran.

On the other hand, it's actually possible and not difficult to set some parameters such as `POINTER_LEN` at configuration.

What do you think about this?